fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -131,6 +131,10 @@ struct ConsistencyCheckWorkload : TestWorkload {
 			try {
 				wait(timeoutError(quietDatabase(cx, self->dbInfo, "ConsistencyCheckStart", 0, 1e5, 0, 0),
 				                  self->quiescentWaitTimeout)); // FIXME: should be zero?
+				if (g_network->isSimulated()) {
+					g_simulator->quiesced = true;
+					TraceEvent("ConsistencyCheckQuiesced").detail("Quiesced", g_simulator->quiesced);
```

Comment:
what's the point of printing g_simulator->quiesced since it will always be true here?

Reply:
Good point. It's not needed indeed.
Since this is a cherrypick, I will not remote this field in this PR.